src/System.Net.Http/src/System/Net/Http/Unix/CurlResponseHeaderReader.cs

```diff
@@ -111,6 +119,18 @@ private static bool ValidHeaderNameChar(byte c)
             return c > ' ' && invalidChars.IndexOf((char)c) < 0;
         }
 
+        internal static bool IsWhiteSpaceLatin1(byte c)
+        {
```

Comment:
I didn't add this, just moved it from below (GitHub doesn't show that well). But i can augment it.